src/components/CartItem.js

```diff
@@ -0,0 +1,61 @@
+import { useContext } from 'react'
+import { CartContext } from '../helper/CartContext'
+
+const CartItem = () => {
+  const addToCart = selectedItem => {
+    const updatedCart = cart.map(item =>
+      item === selectedItem
+        ? { ...item, cartQuantity: item.cartQuantity + 1 }
```

Comment:
like the readability of the the code, maybe could of fitted line 8-11 to less line perhaps ?